src/measure_theory/probability_mass_function/monad.lean

```diff
@@ -107,6 +126,39 @@ begin
   simp [mul_assoc, mul_left_comm, mul_comm]
 end
 
+section measure
+
+variable (s : set β)
+
+lemma to_outer_measure_bind_apply :
+  (p.bind f).to_outer_measure s = ∑' (a : α), (p a : ℝ≥0∞) * (f a).to_outer_measure s :=
+calc (p.bind f).to_outer_measure s
+      = ∑' (b : β), if b ∈ s then (↑(∑' (a : α), p a * f a b) : ℝ≥0∞) else 0
+    : by simp [to_outer_measure_apply, set.indicator_apply]
+  ... = ∑' (b : β), ↑(∑' (a : α), p a * (if b ∈ s then f a b else 0))
+    : tsum_congr (λ b, by split_ifs; simp)
+  ... = ∑' (b : β) (a : α), ↑(p a * (if b ∈ s then f a b else 0))
+    : tsum_congr (λ b, ennreal.coe_tsum $
+        nnreal.summable_of_le (by split_ifs; simp) (bind.summable p f b))
+  ... = ∑' (a : α) (b : β), ↑(p a) * ↑(if b ∈ s then f a b else 0)
+    : ennreal.tsum_comm.trans (tsum_congr $ λ a, tsum_congr $ λ b, ennreal.coe_mul)
+  ... = ∑' (a : α), ↑(p a) * ∑' (b : β), ↑(if b ∈ s then f a b else 0)
+    : tsum_congr (λ a, ennreal.tsum_mul_left)
+  ... = ∑' (a : α), ↑(p a) * ∑' (b : β), if b ∈ s then ↑(f a b) else (0 : ℝ≥0∞)
+    : tsum_congr (λ a, congr_arg (λ x, ↑(p a) * x) $ tsum_congr (λ b, by split_ifs; refl))
+  ... = ∑' (a : α), ↑(p a) * (f a).to_outer_measure s
+    : tsum_congr (λ a, by rw [to_outer_measure_apply, set.indicator])
```

Comment:
The style guide says to put `:` at the end of a line, instead of at the beginning.